src/App.js

```diff
@@ -39,8 +39,13 @@ class App extends Component {
     console.log(this.state);
   }
 
-  onItemClicked = () => {
+  onItemClicked = (type) => {
     // Fill this in for Wave 3!
+    if(type === 'litter') {
```

Comment:
Making this check here removes the `type` variable from its context, which I don't love.